modules/plugin/svg/src/test/java/org/geotools/renderer/lite/DrawTest.java

```diff
@@ -307,6 +307,47 @@ public void testParametricAllValues() throws Exception {
                 1000);
     }
 
+    @Test
+    public void testSvgMark() throws Exception {
+        StreamingRenderer renderer = setupSinglePointRenderer("hospital.sld");
+
+        BufferedImage image = RendererBaseTest.showRender("hospital", renderer, TIME, bounds);
+        ImageAssert.assertEquals(
+                new File("./src/test/resources/org/geotools/renderer/lite/test-data/hospital.png"),
```

Comment:
This paths relative to source make me uncomfortable, I would prefer to see the get as resource stream option ... but well they are perfectly valid and the build is passing :)

Reply:
This approach is used a lot in testing code. If it bothers you enough, raise the issue on the devel list and help fixing all instances, e.g.:

{code}
~/devel/git-gt (svg_mark) $ git grep "src/test/resources" | wc -l
264
{code}

Reply:
First time I notice it, since it appears in 264 it has prove to be correct enough.